### DISCUSSION TITLE
'user/login' Not Found

### DISCUSSION THREAD

**Discussion by lyntux** [magicbug/Cloudlog#1053] [Q&A]
I just got Cloudlog installed, and now I'm going to use it. I try logging in with the default credentials, but I'm getting an error:

Error 404:
https://\<URL-To-Cloudlog\>/index.php/user/login
does not exist.

I'm running Nginx, but I'm not specifically looking for Nginx help. If I can figure out where the problem is, I can troubleshoot the Nginx side of things.

**Comment by resnikov** [comment]
I get the same error and on Nginx too.

**Comment by resnikov** [comment]
HAHA, just fixed it. Its the mode_rewrite for apache. 
see this for more info https://www.digitalocean.com/community/questions/enabling-nginx-mod_rewrite 

but for TLDR
Reaplce this:
 try_files $uri $uri/ =404;

with this:
try_files $uri $uri/ /index.php?$args;

in your server config file in /etc/nginx/sites-available/
